widget-reference/recorder.md

```diff
@@ -0,0 +1,22 @@
+---
+description: >-
+  Recorder widget allows users to record using their microphone, listen to its playback and export the data to a datasource.
+---
+
+# Recorder
+
+## Image/gif of the widget on the canvas with the icon of the widget in the sidebar
+
+## Properties
+
+| Property            | Description                              |
+| :------------------ | :--------------------------------------- |
+| **backgroundColor** | Sets the background color of the widget. |
+| **iconColor**       | Sets the icon color of the widget.       |
+| **isVisible**       | Controls the visibility of the widget.   |
+| **isDisabled**      | Disables input to this widget.           |
+
```

Comment:
Later if we add the `value` as an internal property to provide ref to the data, we should add it here. Please keep a check before closing